cypress/integration/ColorPicker.spec.ts

```diff
@@ -194,32 +194,38 @@ describe('ColorPicker', () => {
       });
 
       context('when a swatch is clicked', () => {
-        const color = '8660d1';
+        const color = '#ff5347';
+
         beforeEach(() => {
-          cy.get(`.wdc-color-picker--color-${color}`).click();
+          getSwatch(color).click();
         });
 
         it('should update the color input to the swatch color', () => {
-          getColorInput().should('have.value', color.toUpperCase());
+          getColorInput().should('have.value', color.slice(1).toUpperCase());
```

Comment:
I don't love the whole `toUpperCase()` portion but I'm also trying to keep this PR small. If I keep tugging here, I may end up changing the stories that we're testing on.